### DISCUSSION TITLE
Report command

### DISCUSSION THREAD

**Discussion by AlternativeSoap** [Zrips/CMI#7578] [Feature Requests]
A report command would be nice to have such as
/report (player) (reason)
/report list

Just those basic commands. I request it because I can't find any stable report plugins and if I find, I need to replace CMI ban, mute etc commands which is unnecessary if those commands could be added.

**Comment by CloudeLecaw** [comment]
might take a look at this one if i could advertice a bit for a report system..

We are using this one on our server and after a bit of configrataion and fighting with setting up the right permissions it works like a charm^^ (we currently use paper 1.19.2)

https://www.spigotmc.org/resources/advanced-mod-request-ticket-management-gui-mysql-notifications-helpop.40528/
